### PR TITLE
feat(husky): update pot files before commit

### PR DESCRIPTION
**Pull request by HendrikThePendric**
Not sure if this is a good idea, but if it is pls hit approve. Also see [this Slack thread](https://dhis2.slack.com/archives/CHENB0272/p1604583935002500) for context.

### CODE REVIEW

**Comment by varl** [comment]
We need to keep in mind that `d2-style` is used in non-app and non-i18n contexts as well, so the default configuration is intentionally generic so that it is compatible with all our JS code.

Option 1 is to customize the `.huskyrc.js` in the apps were `app-platform` is used.

Option 2 is create a new "app"  template for the git hooks in `d2-style`. The same way we have solved the React-eslint vs. Vanilla-eslint setups here.

As this PR is right now, it will not work for us.

**Comment by varl** [comment]
Maybe an "app" here: https://github.com/dhis2/cli-style/blob/master/src/utils/groups.js#L106-L126

**Comment by HendrikThePendric** [comment]
@varl thanks for the comments above. I tried following your instructions to the letter, mainly because that was quicker than getting completely familiarised with how `cli-style` works. Could you review the revised implementation?

File: config/husky-app.local.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+const tasks = arr => arr.join(' && ')
+
+module.exports = {
+    hooks: {
+        ...husky.hooks,
+        'pre-commit': tasks([
+            'd2-style js check --staged',
+            'd2-style text check --staged',
+            'd2-app-scripts i18n',
```
Comment (amcgee) [inline config/husky-app.local.js:12]: This needs to be `d2-app-scripts i18n extract && d2-app-script i18n generate`, I believe

**Review by amcgee — CHANGES_REQUESTED** [review]

File: config/husky-app.local.js (hunk 1 above)
Comment (amcgee) [reply config/husky-app.local.js]: Actually only `extract` is required, technically

**Comment by HendrikThePendric** [comment]
@varl @amcgee this PR has been open for a very long time. I think it's slipped off the radar. It's not really a crucial fix and if I've understood things correctly it will become redundant once we revamp the translation mechanism as a whole. So I'd be happy to just close this issue.

**Comment by amcgee** [comment]
@HendrikThePendric yeah, I think that's fair - sorry it got stale.  This problem should go away when we get rid of source-generated translations.

**Pull request closed without merging**